R/check_crpr.R

```diff
@@ -0,0 +1,131 @@
+.datasets <- new.env(parent = emptyenv())
+
+#' Signal CR Records Followed by PR
```

Comment:
as this will end up living in admiraldev, i'd add a bit extra via an @details section here, e.g. something like: 

Signal Complete Response (CR) tumor response records followed by a subsequent Partial Response (PR)

Note that this is implausible as it would signify the tumor has returned and therefore the subsequent PR should be a Progressive Disease (PD)

Reply:
Is creating a new environment an admiral convention? I am not sure I see a need for this, but I am presuming it is an admiral convention I am missing.

`.datasets <- new.env(parent = emptyenv())`

Reply:
Actually, this needs to get updated for the most recent changes in {admiraldev}. Instead of having each package create its own environemnt for saving these kinds of datasets there's a single one in {admiraldev}. To place a dataset there you'd use `set_dataset()` and to retrieve `get_dataset()`. So `set_crpr_dataset()` would be defiend simply as `set_dataset(crpr_data, "crpr")` instead of `    .datasets$crpr <- crpr_data`. See also here: https://github.com/pharmaverse/admiral/blob/d976949736474e5d2ed012e30a1949307cecca03/R/duplicates.R

Reply:
@bundfussr FYI we discussed at core dev team that any new assertion checks here for this PR could remain in admiralonco for now for this release, and then please make an issue on the backlog for later to move to admiraldev later after our Sept release.

Reply:
@rossfarrugia , I would not move `signal_crpr()` to admiraldev because it is very oncology specific.

Reply:
Will add to the call tomorrow agenda as that would contravene https://pharmaverse.github.io/admiraldev/main/articles/admiraldev.html#scenario-two-admiral-extension. fine to stay in admiralonco for now anyway though so no impact to this PR

Reply:
Thanks, I was not aware of this guidance in admiraldev.